src/main/java/org/sonatype/gradle/plugins/scan/ScanPlugin.java

```diff
@@ -29,11 +31,17 @@
 
   @Override
   public void apply(Project project) {
-    project.getExtensions().create("nexusIQScan", NexusIqPluginExtension.class, project);
+    project.getExtensions().create("nexusIQScan", NexusIqPluginScanExtension.class, project);
     NexusIqScanTask nexusIqScanTask = project.getTasks().create("nexusIQScan", NexusIqScanTask.class);
     nexusIqScanTask.setGroup(SONATYPE_GROUP);
     nexusIqScanTask.setDescription("Scan and evaluate the dependencies of the project using Nexus IQ Server.");
 
+    project.getExtensions().create("nexusIQIndex", NexusIqPluginIndexExtension.class, project);
+    NexusIqIndexTask nexusIqIndexTask = project.getTasks().create("nexusIQIndex", NexusIqIndexTask.class);
+    nexusIqIndexTask.setGroup(SONATYPE_GROUP);
+    nexusIqIndexTask.setDescription("Saves information about the dependencies of a project into a module information "
+        + "file that Sonatype CI tools can use to include these dependencies in a scan.");
```

Comment:
since there might be multiple
```suggestion
    nexusIqIndexTask.setDescription("Saves information about the dependencies of a project into module information "
        + "(module.xml) files that Sonatype CI tools can use to include these dependencies in a scan.");
```